runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/CraftingAction.java

```diff
@@ -74,26 +75,26 @@
 	EMPTY_SACK(ItemID.EMPTY_SACK, 21, 38),
 	SAPPHIRE_NECKLACE(ItemID.SAPPHIRE_NECKLACE, 22, 55),
 	OPAL_BRACELET(ItemID.OPAL_BRACELET, 22, 45),
-	SAPPHIRE_BRACELET(ItemID.SAPPHIRE_BRACELET, 23, 60),
+	SAPPHIRE_BRACELET(ItemID.SAPPHIRE_BRACELET_11072, 23, 60),
 	TIARA(ItemID.TIARA, 23, 52.5f),
 	SAPPHIRE_AMULET_U(ItemID.SAPPHIRE_AMULET_U, 24, 65),
 	UNFIRED_POT_LID(ItemID.UNFIRED_POT_LID, 25, 20),
 	POT_LID(ItemID.POT_LID, 25, 20),
 	JADE_NECKLACE(ItemID.JADE_NECKLACE, 25, 54),
 	WILLOW_BIRD_HOUSE(ItemID.WILLOW_BIRD_HOUSE, 25, 25),
-	DRIFT_NET(ItemID.SMALL_FISHING_NET_6209, 26, 55),
+	DRIFT_NET(ItemID.DRIFT_NET, 26, 55, true),
```

Comment:
This does not need an override--it was only showing up incorrectly as a f2p item because it was entered as a small fishing net rather than a drift net.
```suggestion
	DRIFT_NET(ItemID.DRIFT_NET, 26, 55),
```